src/main/java/org/gitlab4j/api/models/RunnerAccessLevel.java

```diff
@@ -0,0 +1,16 @@
+package org.gitlab4j.api.models;
+
+public enum RunnerAccessLevel {
+    NOT_PROTECTED("not_protected"),
+    REF_PROTECTED("ref_protected");
+
+    private String value;
+
+    RunnerAccessLevel(String value) {
+        this.value = value;
+    }
+
+    public String toValue() {
+        return this.value;
+    }
+}
```

Comment:
Because this is not a shared enum it should be in the RunnersDetail class and follow the convention used elsewhere by using the JacksonJsonEnumHelper:
```
    /** Enum to use for RunnerDetail accessLevel property. */
    public static enum RunnerAccessLevel {

        NOT_PROTECTED, REF_PROTECTED;
        private static JacksonJsonEnumHelper<RunnerAccessLevel > enumHelper = 
            new JacksonJsonEnumHelper<>(RunnerAccessLevel .class);

        @JsonCreator
        public static RunnerAccessLevel forValue(String value) {
            return enumHelper.forValue(value);
        }

        @JsonValue
        public String toValue() {
            return (enumHelper.toString(this));
        }

        @Override
        public String toString() {
            return (enumHelper.toString(this));
        }
    }
```
Using JacksonJsonEnumHelper allows these enums to be consumed without any special logic in the consumer, like looking for null value.